runner.py

```diff
@@ -58,7 +58,7 @@ def configure_gradio_ui_envs() -> None:
     config.init_query_filter()
     config.init_vector_index()
 
-    host = "localhost" if config.dev_config.run_on_localhost else None
+    host = "localhost" if config.dev_config.run_on_localhost else "0.0.0.0"  # noqa S104
```

Comment:
`None` was previously transformed to `127.0.0.1`. Is this change intended?

Reply:
was where?

Reply:
in `uvicorn.run` when the `host=None`

Reply:
when it is *not specified*, it is 127.0.0.1, not when we pass *None* in string param